src/Mvc/Mvc.ViewFeatures/src/Microsoft.AspNetCore.Mvc.ViewFeatures.csproj

```diff
@@ -26,4 +26,11 @@ Microsoft.AspNetCore.Mvc.ViewComponent</Description>
     <Reference Include="Microsoft.AspNetCore.Components.Authorization" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Include="..\..\..\Shared\Components\ComponentDescriptorSerializer.cs" />
```

Comment:
I think we some msbuild variables that point to this? Like `SharedRoot`?

Reply:
I switched this to use RepoRoot, SharedRoot is a thing of the past I believe. Arcade doesn't have that concept.

Reply:
done